bool segComplete

uint64 seg_number

float64 progress_made